index.html

```diff
@@ -3761,6 +3767,66 @@ <h3><dfn>Get Window Handles</dfn></h3>
 </aside>
 </section> <!-- /Get Window Handles -->
 
+<section>
+<h3><dfn>Create Window</dfn></h3>
+
+<table class="simple jsoncommand">
+ <tr>
+  <th>HTTP Method</th>
+  <th>URI Template</th>
+ </tr>
+ <tr>
+  <td>GET</td>
+  <td>/session/{<var>session id</var>}/window/new</td>
+ </tr>
+</table>
+
+<p>Create a new top level browsing context.
+
+<p>The <a>remote end steps</a> are:
+
+<ol>
+ <li><p>If there is an active <a>user prompt</a>, that prevents the
+  creation of another <a>top-level browsing context</a>,
+  return <a>error</a> with <a>error code</a> <a>unexpected alert open</a>.
+
+ <li><p>If the implementation does not support creating new top-level
+  browsing contexts, return <a>error</a> with <a>error code</a>
+  <a>unsupported operation</a>.
+
+ <li><p>Let <var>type hint</var> be the result of <a>getting the property</a>
+  "<code>type</code>" from the <var>parameters</var> argument.
+
+ <li><p>Create a new <a>top-level browsing context</a>. If <var>type
+  hint</var> has the value "<code>tab</code>" and the implementation
+  supports multiple browsing context in the same OS window, the new
+  browsing context should share an OS window with the <a>current
+  browsing context</a>, otherwise the created browsing context should be in a
+  new OS window.
+
+ <li><p>Let <var>handle</var> be the
+  associated <a>window handle</a> of the newly created window.
+
+ <li><p>Let <var>type</var> be "<code>tab</code>" if the newly created
+  window shares an OS-level window with the <a>current browsing
+  context</a>, or "<code>window</code>" otherwise.
+
+ <li><p>Let <var>result</var> be a new JSON <a>Object</a> initialised with:
+
+ <dl>
+  <dt>"<code>handle</code>"
+  <dd>The value of <var>handle</var>.
+
+  <dt>"<code>type</code>"
+  <dd>The value of <var>type</var>.
+
+ </dl>
+
+ <li><p>Return <a>success</a> with data <a>result</a>.
```

Comment:
Not if it only allows opening `about:blank` since that loads synchronously. And I've made it explicit that the new window has to initially contain `about:blank`.

Reply:
Oh, I see. Thanks for the clarification. I assume the synchronous load is equal for all browsers?

Reply:
It isn't possible to observe the state where `about:blank` isn't loaded in any browser. They *may* differ on whether there are events relating to the initial `about:blank` document loading or not, but that isn't visible through this API so I'm not concerned about it here.